filebeat/config/config.go

```diff
@@ -88,11 +88,11 @@ type JSONConfig struct {
 }
 
 type MultilineConfig struct {
-	Negate   bool   `config:"negate"`
-	Match    string `config:"match"`
-	MaxLines *int   `config:"max_lines"`
-	Pattern  string `config:"pattern"`
-	Timeout  string `config:"timeout"`
+	Negate   bool           `config:"negate"`
+	Match    string         `config:"match"       validate:"required"`
```

Comment:
Will this only hit "required" if multiline is set itself? Means it can be that multiline is not set at all ...